docs/modules/exploratory.rst

```diff
@@ -10,4 +10,5 @@ and visualize functional data.
    
    exploratory/visualization
    exploratory/depth
-   exploratory/outliers
\ No newline at end of file
+   exploratory/outliers
+   exploratory/fpca
```

Comment:
FPCA is not a main category. It should be in preprocessing/dim_reduction/projection in my opinion

Reply:
As it is a technique used for dimension reduction I see what you mean. It is basically a special type of projection. I agree with you and I'll soon submit another commit with file locations changed.

Reply:
And also, there are some FPCA code already in the same directory, but I believe that the FPCA done there is incorrect as it does not take into account the weight matrix. Should I remove that file?

Reply:
Yes, remove it.